templates/theme-topic/_base.scss

```diff
@@ -0,0 +1,8 @@
+/* Theme/topic decorative image */
+.provisional.gc-topic-bg div {
```

Comment:
This will be more a provisional component

Reply:
@duboisp This is debatable whether or not this part should be in its own component as its only use case is on the Intitutional landing page (ILP) and shouldn't be used elsewhere unless we come up with another use case, right? Or if it should be in its own folder, then it should be specified in the documentation that it is currently only used on the ILP.

Reply:
We would need to find an efficient way to manage those CSS. I don't want them to be forgotten neither be usable by other template. I am thinking to lock them down to use a specific CSS body class.

Reply:
Is the intend is to lock down this style as a unique style for this template. If so I am expecting to see something like `body.gc-topic-gc`. I don't want this to be usable by other template that would not be properly tracked.